docs/how-to-guides/integrating-autoware/creating-maps/pointcloud-map-downsampling/index.md

```diff
@@ -0,0 +1,63 @@
+# Pointcloud map downsampling
+
+## Overview
+
+In some cases, for example,
+when your created point cloud map is either too dense or too large (i.e., exceeding 300 MB),
+you may want to downsample it for improved computational and memory efficiency.
```

Comment:
Maybe it's simpler to phrase it this way
```suggestion
When your created point cloud map is either too dense or too large (i.e., exceeding 300 MB),
you may want to downsample it for improved computational and memory efficiency.
```

Reply:
Done :+1: